src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -203,6 +203,7 @@ QgsPostgresConn::QgsPostgresConn( const QString &conninfo, bool readOnly, bool s
   , mOpenCursors( 0 )
   , mConnInfo( conninfo )
   , mGeosAvailable( false )
+  , mProjAvailable( false )
```

Comment:
This was because there was an omission in debugging when the original (unknown to me) programmer queried for the database capabilities; he meant to output the PostgreSQL proj version in debug output, but missed that step, which caused a Qt warning.